src/org/labkey/test/components/glassLibrary/grids/HeatMap.java

```diff
@@ -0,0 +1,211 @@
+package org.labkey.test.components.glassLibrary.grids;
```

Comment:
This package should match the location of the product component in `labkey-ui-components`
```suggestion
package org.labkey.test.components.glassLibrary.heatmap;
```

Reply:
done